CityOfDetroit/bloom
# Remove listing fields

## Issue

- Closes #988
- Addresses #988

## Description

Removes sections from a listing management panel.
For sections:
- How is the review order determined? (FCFS, Lottery)
- Is this a referral?

I've also removed related fields (they were visible then a boolean option was selected).

## Type of change


## How Can This Be Tested/Reviewed?

1. Check if the sections listed below don't exist:
- Preferences
- Programs
- How is the review order determined? (FCFS, Lottery)
- Is this a referral?

2. Try to save an application as a draft
3. Try to publish the application

## Checklist:
